src/main/java/seedu/address/model/common/Category.java

```diff
@@ -0,0 +1,4 @@
+package seedu.address.model.common;
+
+public class Category {
```

Comment:
Is it possible to add a constructor with some parameters instead of just an empty class?